README.md

```diff
@@ -40,38 +40,38 @@ This vagrant box aims to make it dead simple to start a hashistack and emulate h
 ## Content
 
 1. [Description - what & why](#description---what--why)
-    1. [Submodules](#submodules)
-    2. [Services](#services)
-    3. [Why does this exist?](#why-does-this-exist)
-    4. [Installed stack](#installed-stack)
-    5. [Versions](#versions)
+   1. [Submodules](#submodules)
+   2. [Services](#services)
+   3. [Why does this exist?](#why-does-this-exist)
+   4. [Installed stack](#installed-stack)
+   5. [Versions](#versions)
 2. [Install prerequisites](#install-prerequisites)
-    1. [General requirements](#general-requirements)
-        1. [Proxy](#proxy)
-    2. [Linux requirements](#linux-requirements)
-    3. [MacOS requirements](#macos-requirements)
-    4. [Windows requirements](#windows-requirements)
-        1. [GNU Make and Git Bash](#GNU-Make-and-Git-Bash)
-        2. [Installation](#installation)
+   1. [General requirements](#general-requirements)
+      1. [Proxy](#proxy)
+         1. [MacOS](#macos)
+   2. [Linux requirements](#linux-requirements)
+   3. [MacOS requirements](#macos-requirements)
```

Comment:
```suggestion
   1. [General requirements](#general-requirements)
      1. [Proxy](#proxy)
   2. [Linux requirements](#linux-requirements)
   3. [MacOS requirements](#macos-requirements)
      1. [Certificates](#certificates)
```
Shouln't it be something like this instead?

Reply:
Are you asking if this information about certificates should be under `MacOS requirements`? If so, no, I don't think so. This is a transparent proxy problem, and is not a MacOS requirement, it's a proxy requirement

Reply:
Ok, I think I've misunderstood then. I agree with what you have done already.